### PR TITLE
Add Factory Reset

### LINKED ISSUE #362: Add a "factory reset" for the config.

### PR DESCRIPTION
**Pull request by linknum23**
Fixes #362

### CODE REVIEW

**Comment by linknum23** [comment]
Needs testing!!

**Comment by linknum23** [comment]
Tested and working!

Check out the wording on the interface changes and try it out for yourself.

File: amplipi/app.py [hunk 1]
```diff
@@ -149,6 +149,11 @@ def load_config(config: models.Status, ctrl: Api = Depends(get_ctrl)) -> models.
   ctrl.reinit(settings=ctrl._settings, change_notifier=notify_on_change, config=config)
   return ctrl.get_state()
 
+@api.post('/api/factory_reset', tags=['status'])
+def load_factory_config(ctrl: Api = Depends(get_ctrl)) -> models.Status:
+  """ Load the "factory" configuration (and return the configuration loaded). This will overwrite the current configuration so it is advised to save the previous config from. """
```
Comment (Lohrer) [inline amplipi/app.py:154]: save the previous config from what? I'm in suspense!

**Review by Lohrer — APPROVED** [review]
The description is a bit wordy but it's all important info so I don't really know how to shorten it. Looks good!
